Pipfile

```diff
@@ -21,7 +21,7 @@ sqlalchemy = "*"
 maya = "*"
 flask = "*"
 # Third-Party Ethereum
-py-evm = "*"
+py-evm = "==0.2.0a39"
```

Comment:
I guess pinning py-evm to this specific version is intended, right?
Also, apart from creating a Pipfile.lock, it is necessary to create the `requirements.txt` and `dev-requirements.txt` files: 
```
pipenv lock --three --clear --pre --requirements > requirements.txt
pipenv lock --three --dev --clear --pre --requirements > dev-requirements.txt
```